src/automyte/config/config.py

```diff
@@ -57,13 +60,35 @@ def set_vcs(self, **kwargs):
         return self
 
     @classmethod
-    def _load_from_config_file(cls, config_file_path: str | Path = "./automyte.cfg") -> ConfigParams:
+    def _load_from_config_file(
```

Comment:
I think we have 120 chars line length
are you sure you're sourcing your linter settings from `pyproject.toml`?